lib/logthrdestdrv.h

```diff
@@ -63,6 +65,7 @@ struct _LogThreadedDestWorker
   gint batch_size;
   gint rewound_batch_size;
   gint retries_counter;
+  gint seq_num;
```

Comment:
I think this should remain gint32: step_sequence_number_atomic returns with gint32.

Reply:
good catch.